cmd/lncli/commands_test.go

```diff
@@ -64,7 +64,14 @@ func TestParseChanPoint(t *testing.T) {
 	}
 	for _, tc := range testCases {
 		cp, err := parseChanPoint(tc.channelPoinStr)
-		require.Equal(t, tc.err, err)
+
+		require.Equal(t, tc.err == nil, err == nil)
```

Comment:
Since we're wrapping errors now, I think we can just do `require.ErrorIs(t, err, tc.err)` and update the two test cases to use the actual cause as the error:
```
		}, {
			"24581424081379576b4a7580ace91db10925d996a2a8d45c8034" +
				"3a5a467dc0bc:string",
			true,
			0,
			strconv.ErrSyntax,
		}, {
			"not_hex:0",
			true,
			0,
			hex.InvalidByteError('n'),
		},
```